Android/build.gradle

```diff
@@ -18,27 +18,28 @@ dependencies {
 	implementation("net.twisterrob.lib:twister-lib-android-slf4j")
 	implementation("net.twisterrob.lib:twister-lib-android-widgets")
 	implementation("net.twisterrob.lib:twister-lib-android-stringers")
+
 	implementation("net.twisterrob.lib:twister-lib-android-about")
+	implementation("net.twisterrob.lib:twister-lib-android-color_picker")
 	implementation("net.twisterrob.lib:twister-lib-android-settings")
 
 	implementation(project(':Shared')) {
 		exclude module: 'android-polyfill'
 	}
 	implementation("com.google.android.gms:play-services-maps:${VERSION_MAPS}")
 	implementation("com.google.android.gms:play-services-places:${VERSION_MAPS}")
-	implementation("com.android.support:support-v4:${VERSION_SUPPORT}")
-	implementation("com.android.support:appcompat-v7:${VERSION_SUPPORT}")
-	implementation("com.android.support:design:${VERSION_SUPPORT}")
+	implementation("com.google.code.findbugs:jsr305:${VERSION_JSR305}")
 	implementation("com.github.bumptech.glide:glide:${VERSION_GLIDE}")
 
 	apply from: "${rootDir}/gradle/testCompile.gradle", to: project
 }
 
 android {
-	compileSdkVersion 28
+	namespace = "net.twisterrob.blt.android"
+	compileSdk = 34
```

Comment:
Necessary for all the dependencies needing resources in new versions. This triggers a ton of deprecation, which are turned off for now.